packages/@sanity/desk-tool/src/panes/document/statusBar/sparkline/PublishStatus.tsx

```diff
@@ -1,25 +1,24 @@
 /* eslint-disable complexity */
 import {useTimeAgo} from '@sanity/base/hooks'
-
 import {Box, Button, Flex, Stack, Text, Tooltip} from '@sanity/ui'
 import {PlayIcon, PublishIcon} from '@sanity/icons'
 import React from 'react'
 import styled from 'styled-components'
-import {IconBadge} from './IconBadge'
 
 interface PublishStatusProps {
   disabled: boolean
   lastPublished?: string
   lastUpdated?: string
   liveEdit: boolean
+  collapsed?: boolean
```

Comment:
Nitpick: would be great to alphabetize these props 🙏

Reply:
Added the plugin you mentioned, thank you for the tip :)